javascript/src/main/pages/SearchHistory/SearchHistory.js

```diff
@@ -18,16 +19,39 @@ const SearchHistory = () => {
         text: 'Timestamp' ,
         sort: true
     }];
-
+    
+ 
+        
     const { getAccessTokenSilently: getToken } = useAuth0();
     
-    const { data: usersearch } = useSWR(["/api/members/searchhistory/allusersearches", getToken], fetchWithToken);
+    const { data: usersearch } = useSWR(["/api/members/searchhistory/specificuser", getToken], fetchWithToken);
+    
+     const { data: roleInfo } = useSWR(
+        ["/api/myRole", getToken],
+        fetchWithToken
+      );
+      const isAdmin = roleInfo && roleInfo.role.toLowerCase() === "admin";
+      
+   
+/*    if(isAdmin){
+         return (
+            <div>
+                <h1>Show Search History</h1>
+                <Link to="/member/searchhistory/alluser" className="btn btn-primary">ViewAllUsers</Link>
+
+                
+        <BootstrapTable keyField='id' data={usersearch || []} columns={columns} />
+            </div>
+        );
+    }*/
```

Comment:
Can we remove this commented block of code?

Reply:
Have delete the comment